### ISSUE TITLE
Cannot create table

### ISSUE THREAD

**Issue by ispiropoulos** [TDAbboud/PhotosGalleryApp#1]
``` swift
2015-01-26 18:43:01.636 Photos Gallery App[88002:23865681] 
Folder "App Folder" does not exist
Creating now...
2015-01-26 18:43:01.637 Photos Gallery App[88002:23866166] Creation of folder -> Error!
ViewWillAppear
fatal error: unexpectedly found nil while unwrapping an Optional value
(lldb) 
```

**Comment by fxpage** [comment]
maybe you didn't set the outlet of imageview in the photoCell of collectionview

**Comment by ghost** [comment]
maybe you didn't set the outlet of imageview of PhotoCell in collectionview.

**Comment by jefferyleo** [comment]
Having the same problem!

**Comment by tabboud** [comment]
This should be resolved by 0cde437ff9bc2a2de6ab2cb1bbe7472434323154.

Feel free to open it if the problem persists.
